dev/Expander/Expander.xaml

```diff
@@ -0,0 +1,200 @@
+﻿<!-- Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT License. See LICENSE in the project root for license information. -->
+<ResourceDictionary
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:local="using:Microsoft.UI.Xaml.Controls"
+    xmlns:contract7Present="http://schemas.microsoft.com/winfx/2006/xaml/presentation?IsApiContractPresent(Windows.Foundation.UniversalApiContract,7)">
+    <Style TargetType="local:Expander">
```

Comment:
The default style should be named with a Key and then exposed with the default.
```suggestion
    <Style TargetType="local:Expander" BasedOn="{StaticResource DefaultExpanderStyle}"/>
    
    <Style x:Key="DefaultExpanderStyle" TargetType="local:Expander">
```

See ComboBox for instance:

https://github.com/microsoft/microsoft-ui-xaml/blob/da3964ea1f8d79032975d3820d3d269f200bf03b/dev/ComboBox/ComboBox_themeresources.xaml#L391-L393

Otherwise you can't use the default style as a basis to easily override it. ☹

I also suggest adopting splitting out the `ControlTemplate` part for similar reasons (create a completely new style but with the same control template). We do this in the Toolkit, but I know that's not the standard practice in WinUI currently.